test/tests/server.js

```diff
@@ -53,7 +53,7 @@ describe('Server', () => {
     it('GET request to bundle file', (done) => {
       request(app).get('/public/bundle.js')
         .expect('Content-Type', 'application/javascript; charset=UTF-8')
-        .expect('Content-Length', '2839')
```

Comment:
please use a conditional based on the webpack version number to set these values.